src/pyOpenMS/pxds/IDFilter.pxd

```diff
@@ -19,82 +19,146 @@ cdef extern from "<OpenMS/FILTERING/ID/IDFilter.h>" namespace "OpenMS":
+                #   -----
+                #   :param groups: Input/output protein groups
+                #   :param hits: Available protein hits (all others are removed from the groups)
+                #   :return: Returns whether the groups are still valid (which is the case if only whole groups, if any, were removed).
 
-        void removeEmptyIdentifications(libcpp_vector[PeptideIdentification]& ids) nogil except +
-        void removeEmptyIdentifications(libcpp_vector[ProteinIdentification]& ids) nogil except +
+        void removeEmptyIdentifications(libcpp_vector[PeptideIdentification]& ids) nogil except + # wrap-doc:Removes peptide or protein identifications that have no hits in them
+        void removeEmptyIdentifications(libcpp_vector[ProteinIdentification]& ids) nogil except + # wrap-doc:Removes peptide or protein identifications that have no hits in them
 
-        void filterHitsByScore(libcpp_vector[PeptideIdentification]& ids, double threshold_score) nogil except +
-        void filterHitsByScore(libcpp_vector[ProteinIdentification]& ids, double threshold_score) nogil except +
+        void filterHitsByScore(libcpp_vector[PeptideIdentification]& ids, double threshold_score) nogil except + # wrap-doc:Filters peptide or protein identifications according to the score of the hits. Only peptide/protein hits with a score at least as good as 'threshold_score' are kept. Score orientation (are higher scores better?) is taken into account
+        void filterHitsByScore(libcpp_vector[ProteinIdentification]& ids, double threshold_score) nogil except + # wrap-doc:Filters peptide or protein identifications according to the score of the hits. Only peptide/protein hits with a score at least as good as 'threshold_score' are kept. Score orientation (are higher scores better?) is taken into account
 
-        void keepNBestSpectra(libcpp_vector[PeptideIdentification]& peptides, Size n) nogil except +
+        void keepNBestSpectra(libcpp_vector[PeptideIdentification]& peptides, Size n) nogil except + # wrap-doc:Filter identifications by "N best" PeptideIdentification objects (better PeptideIdentification means better [best] PeptideHit than other)
 
-        void keepNBestHits(libcpp_vector[PeptideIdentification]& ids, Size n) nogil except +
-        void keepNBestHits(libcpp_vector[ProteinIdentification]& ids, Size n) nogil except +
+        void keepNBestHits(libcpp_vector[PeptideIdentification]& ids, Size n) nogil except + # TODO
+        void keepNBestHits(libcpp_vector[ProteinIdentification]& ids, Size n) nogil except + # TODO
 
         void filterHitsByRank(libcpp_vector[PeptideIdentification]& ids, Size min_rank, Size max_rank) nogil except +
+            # wrap-doc:
+                #   Filters peptide or protein identifications according to the ranking of the hits
+                #   -----
+                #   The hits between 'min_rank' and 'max_rank' (both inclusive) in each ID are kept.
```

Comment:
remove dot (also in the sentences below)

Reply:
I think it's fine if we keep the dots in the multiline statements.

Reply:
Yes, but then this has to be consistent.

Reply:
fixed